minvws/nl-kat-coordination
# Update filter unit and integration tests

### Changes

Added some additional test to make sure nested jsonb queries perform as expected

### Issue link

N/A

### QA notes

Just some additional tests. Can check if the tests run by checking out the branch and issuing:
`make itest` 

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
